src/provider/model/AppWindow.ts

```diff
@@ -126,11 +126,11 @@ export abstract class AbstractAppWindow implements AppWindow {
     }
 
     public get channelContextListeners(): ReadonlyArray<ChannelId> {
-        return Object.keys(this._channelContextListeners);
+        return Array.from(this._channelContextListeners.values());
```

Comment:
What is the value of this change? Not sure I've fully understood what's happening here.

Reply:
`Object.keys` doesn't work as expected for `Map` (it's basically just another class, rather than anything `Object.keys` will give a meaningful result for), and was always returning an empty array

Reply:
Ah, ok. Something leftover from when we switched from object to Map I guess.